src/content/en/fundamentals/primers/service-workers/where-they-live.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: shared/narrow
+published: false
+title: "Where does the code live?"
+description: "A basic service worker implementation requires code in two places."
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 3
+translation_priority: 0
+authors:
+- josephmedley
+---
+
+<p class="intro">
+  A basic service worker implementation requires code in two places.
+</p>
+
+ 
+### Clients
+
+A client is a web page that uses a service worker. One or more clients 
+can use the same service worker. All they need do is implement the 
+registration code, which we'll look at in the next section.
+
+### Service worker script
+
+This is a background script that acts as a network proxy for one or
+more clients. Though its name implies activity, you can almost think of a
+service worker as a passive player, one that only runs when one of its
+event handlders are fired. It can't even manipulate its clients' DOM's.
```

Comment:
http://www.dailywritingtips.com/when-to-form-a-plural-with-an-apostrophe/ claims to quote the _Chicago Manual of Style_ and suggests that it should be DOMs:

> DO NOT use an apostrophe to form the plural of capital letters used as words, abbreviations that contain no interior periods, and numerals used as nouns:
> the three Rs.
> the 1990s
> lengthy URLs

If you follow a style guide that suggests otherwise, I'm happy to defer to that.